core/CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [2.9.3] (21st October 2025)
+
+### Added
+- Support using both Tropic and Optiga to protect PIN.  [#5845]
```

Comment:
This should probably be `[T3W1]`.

Also maybe a good time to start CHANGELOG.T3W1.md.